src/arrayMethodSort.js

```diff
@@ -3,9 +3,41 @@
 /**
  * Implement method Sort
  */
+
+const compareAsString = (a, b) => {
+  const aToString = String(a);
+  const bToString = String(b);
+
+  if (aToString > bToString) {
+    return 1;
+  } else if (aToString === bToString) {
+    return 0;
+  }
```

Comment:
If you write `return` inside `if` , `else` is redundant.
```suggestion
  if (aToString > bToString) {
    return 1;
  } 
  
  if (aToString === bToString) {
    return 0;
  }
```